dandi/bids_validator_xs.py

```diff
@@ -519,6 +528,7 @@ def write_report(
         else:
             f.write("All mandatory BIDS files were found.\n")
         f.close()
+    lgr.info(f"BIDS validation log written to {report_path}")
```

Comment:
Please don't use f-strings for logging, use %s and place value as argument. Rationale could be found in some prior PRs